src/server/mappers/TagMapper.ts

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable class-methods-use-this, lines-between-class-members, no-dupe-class-members */
+import { injectable } from 'inversify'
+import { StorableTag } from '../repositories/types'
+import { TagType } from '../models/tag'
+import { Mapper } from './Mapper'
+
+@injectable()
+export class TagMapper implements Mapper<StorableTag, TagType> {
+  persistenceToDto(tagType: TagType): StorableTag
+  persistenceToDto(tagType: TagType | null): StorableTag | null {
```

Comment:
Just wondering why we need this class `TagMapper` if we don't permute the data. 

Here, we're only checking for existence and returning the object as is if it exists, otherwise returning null. We could skip this mapper class and return the object directly since we don't permute it.